lib/crypto/sr25519/sr25519_test.go

```diff
@@ -5,8 +5,12 @@ package sr25519
 
 import (
 	"crypto/rand"
+	"errors"
+
```

Comment:
nit I don't think there should  be an empty line here. Although fine by me if the  CI allows it

```suggestion
```